packages/shared/scope-hoisting/src/link.js

```diff
@@ -217,35 +217,33 @@ export function link({
     if (mod.meta.isCommonJS && originalName === 'default') {
       let name = getName(mod, '$interop$default');
       if (!path.scope.getBinding(name)) {
-        // Hoist to the nearest path with the same scope as the exports is declared in
-        let binding = path.scope.getBinding(
-          assertString(mod.meta.exportsIdentifier),
+        let binding = nullthrows(
```

Comment:
@mischnic I believe this is a same assumption to make. Let me know if you know of a case where the binding may not exist.

Reply:
Yes, in the `hasAsset` case, it obviously is. And `addBundleImport` creates a binding:
https://github.com/parcel-bundler/parcel/blob/52f0cbbde1ffc85986781c8590b5cf19fcc34e7d/packages/shared/scope-hoisting/src/link.js#L329-L336